pennylane/templates/embeddings/displacement.py

```diff
@@ -88,6 +46,41 @@ def DisplacementEmbedding(features, wires, method="amplitude", c=0.1):
         ValueError: if inputs do not have the correct format
     """
 
-    wires = Wires(wires)
-    pars = _preprocess(features, wires, method, c)
-    broadcast(unitary=qml.Displacement, pattern="single", wires=wires, parameters=pars)
+    num_params = 1
+    num_wires = AnyWires
+    par_domain = "A"
+
+    def __init__(self, features, wires, method="amplitude", c=0.1, do_queue=True):
+
+        shape = qml.math.shape(features)
+        constants = [c] * shape[0]
+        constants = qml.math.convert_like(constants, features)
```

Comment:
This line is new: Tensorflow did not like to stack a tensor with a list. A sign that the new tests are better.